src/style/helpers.scss

```diff
@@ -126,5 +126,10 @@
 
 .login-input {
   max-width: 450px;
-
 }
+
+/***********Context Menu**************/
+.dropdown-toggle:after{
+  /* overwrites bootstrap dropdown-toggle:after */
```

Comment:
This also remove it everywhere, so add a custom class to make it scoped only on this dropdown

Reply:
i fixed it